pkg/Microsoft.ML.TimeSeries/Microsoft.ML.TimeSeries.nupkgproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Pack">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageDescription>Microsoft.ML.TimeSeries contains ML.NET Time Series prediction algorithms. Uses Intel Mkl.</PackageDescription>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Content Include="$(PackagesDir)\mlnetmkldeps\$(MlNetMklDepsPackageVersion)\LICENSE.txt" Pack="true" PackagePath=".\NOTICE.txt" /> 
+    <Content Include="..\common\CommonPackage.props" Pack="true" PackagePath="build\netstandard2.0\$(MSBuildProjectName).props" />
```

Comment:
><Content Include="..\common\CommonPackage.props" Pack="true" PackagePath="build\netstandard2.0\$(MSBuildProjectName).props" /> [](start = 4, length = 126)

Can we have comment, what this is for?

Reply:
Redistribution notice.

Reply:
I'm not talking about .\Notice.txt, I'm talking about  ..\common\CommonPackage.props

---
In reply to: [221393266](https://github.com/dotnet/machinelearning/pull/977#discussion_r221393266) [](ancestors = 221393266)